docs/jwst/introduction.rst

```diff
@@ -491,6 +464,49 @@ There are many pre-defined pipeline modules for processing
 data from different instrument observing modes through each of the 3 stages
 of calibration. For all of the details see :ref:`pipelines`.
 
+.. _pipeline_step_suffix_definitions:
+
+Pipeline/Step Suffix Definitions
+--------------------------------
+
+However the output file name is determined (:ref:`see above
+<intro_output_file_discussion>`), the various stage 1, 2, and 3 pipeline modules
+will use that file name, along with a set of predetermined suffixes, to compose
+output file names. The output file name suffix will always replace any existing
```

Comment:
Will it really "always replace *any* existing suffix" or will it only do that if the existing suffix is one of the known suffixes in the following list?